.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.25.0
+  - 1.26.0
```

Comment:
Why?

Reply:
As we discussed, 1.25 will require boxes here: https://github.com/tox-rs/tox-node/blob/master/src/main.rs#L190-L192
Besides that I used `impl trait` that is also was stabilized in 1.26.

Reply:
As we discussed, it should be noted in README

Reply:
But we don't have mention of rust version in readme of `tox-node`. Shall I add it?

Reply:
Yes please